tool/paths/paths_windows.go

```diff
@@ -15,3 +20,26 @@ const (
 	WizardBinaryName     = "amazon-cloudwatch-agent-config-wizard.exe"
 	AgentStartName       = "amazon-cloudwatch-agent-ctl.ps1"
 )
+
+func init() {
+	programFiles := os.Getenv("ProgramFiles")
+	var programData string
+	if _, ok := os.LookupEnv("ProgramData"); ok {
+		programData = os.Getenv("ProgramData")
+	} else {
+		// Windows 2003
+		programData = filepath.Join(os.Getenv("ALLUSERSPROFILE"), "Application Data")
+	}
+
+	AgentRootDir := programFiles + AgentDir
```

Comment:
nit: Shouldn't this be a `filepath.Join` as well?

Reply:
The problem was that `AgentDir` is a constant and it has leading and trailing `\\`.
And it is used elsewhere, so I didn't want to change it.

And I cannot use filepath.Join to initialize a constant.

Reply:
I went ahead and changes this since filepath.Join will ignore extra file path separators.